api/src/pcapi/routes/serialization/offerers_serialize.py

```diff
@@ -49,6 +53,7 @@ def from_orm(cls, venue: offerers_models.Venue) -> "GetOffererVenueResponseModel
             )
             or venue.hasPendingBankInformationApplication
         )
+        venue.hasCreatedOffer = False if venues_with_offers is None else venue.id in venues_with_offers
```

Comment:
@AmineLouveau : ... et donc ici, puisqu'on sait qu'on a forcément un itérable (set ou list) et jamais None, on peut simplifier : 

```diff
- venue.hasCreatedOffer = False if venues_with_offers is None else venue.id in venues_with_offers
+ venue.hasCreatedOffer = venue.id in venues_with_offers
```